### ISSUE TITLE
[timeseries] fine-grained options on which fields have changed per step?

### ISSUE THREAD

**Issue by dglaeser** [dglaeser/gridformat#234]
The `vtk-hdf` format has an offset mechanism with which to define the values per time step. This allows to reuse data from older time steps in case nothing changed. Currently, the `vtk-hdf` writer takes options in the constructor stating if the grid and/or metadata are constant throughout the simulation. In this case, the grid is written only once.

However, some transient simulations may update the grid only at certain points in time. If we had a mechanism to specify this for individual time steps, such output could be written in a maximally space-efficient manner.

Even individual fields could be flagged as constant with this mechanism, such that the data can be reused in other time steps...

**Comment by dglaeser** [comment]
Closing this because it is a duplicate of #200
